apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -293,6 +309,16 @@ export const WorkoutEditor = ({
             Use without saving
           </Button>
 
+          {canBeDeleted && (
+            <Button
+              onClick={() => setShowDeleted(true)}
```

Comment:
Can we fire a toast instead? I guess that is how we have done it for most of the other UX feedback

Reply:
You mean the success toast here and the error toast still in the function?

Reply:
Sorry; skimmed through the code too quickly, and the name threw me off; could it be named something like `setShowDeleteDialog` instead? 😅

Reply:
yes, that name was a temp name gone perm :))

Reply:
And a success toast is probably nice (?) instead of the info – we mostly use the success toast (?)

Reply:
sure (?) 🤠